rules/windows/network_connection/net_connection_win_imewdbld.yml

```diff
@@ -1,12 +1,16 @@
-title: Download a File with IMEWDBLD.exe
+title: Network Connection Initiated By IMEWDBLD.EXE
 id: 8d7e392e-9b28-49e1-831d-5949c6281228
+related:
+    - id: 863218bd-c7d0-4c52-80cd-0a96c09f54af
+      type: derived
 status: test
-description: Use IMEWDBLD.exe (built-in to windows) to download a file
+description: Detects network connections initiated by IMEWDBLD. Which might indicate potential abuse to download arbitrary files via this utility
```

Comment:
```suggestion
description: Detects network connections initiated by IMEWDBLD. This might indicate potential abuse to download arbitrary files via this utility
```